packages/codemirror-extension/schema/plugin.json

```diff
@@ -26,6 +26,36 @@
       "command": "codemirror:select-next-occurrence",
       "keys": ["Accel Shift D"],
       "selector": ".cm-content"
+    },
+    {
+      "command": "codemirror:fold-current",
+      "keys": ["Accel Shift ["],
+      "selector": ".cm-content"
+    },
+    {
+      "command": "codemirror:unfold-current",
+      "keys": ["Accel Shift ]"],
+      "selector": ".cm-content"
+    },
```

Comment:
This breaks switching tabs:

https://github.com/jupyterlab/jupyterlab/blob/a223935333b987f07da74634d479cafbd220b830/packages/application-extension/schema/commands.json#L5-L14

I would drop these shourtcuts.